example/config/scripts/density_temperature.py

```diff
@@ -124,6 +135,12 @@ def make_single_image(
 
     arguments = ScriptArgumentParser(description="Basic density-temperature figure.")
 
+    # set additional argument defaults if not specified 
+    if not hasattr(arguments, "cmap_order"):
+        arguments.cmap_order = 'regular'
```

Comment:
Please don't use single quotes - always double - throughout. I think `black` should take care of this?

Reply:
ok